pom.xml

```diff
@@ -42,10 +42,13 @@
         <commons-lang.version>2.6</commons-lang.version>
         <node.version>v7.1.0</node.version>
         <yarn.version>v0.19.1</yarn.version>
+        <junit.version>4.12</junit.version>
 
         <ddf.scm.connection.url />
         <snapshots.repository.url />
         <reports.repository.url />
+
+        <project.report.output.directory>project-info</project.report.output.directory>
```

Comment:
This was missing causing the generated directory names to use the replacement variable.